Dockerfile

```diff
@@ -11,52 +11,18 @@ RUN apt-get update -y
 RUN apt-get upgrade -y
 
 # Install Scrapy and Textract dependencies
-RUN apt-get install -y python-dev \
+RUN apt-get install -y python3 \
     python3-dev \
-    libxml2-dev \
-    libxslt1-dev \
-    antiword \
-    unrtf \
-    poppler-utils \
-    pstotext \
-    tesseract-ocr \
-    flac \
-    ffmpeg \
-    lame \
-    libmad0 \
-    libsox-fmt-mp3 \
-    sox \
-    libjpeg-dev \
-    swig \
-    python-virtualenv \
-    python-pip \
-    idle-python2.7 \
-    libgle3 \
-    zlib1g-dev \
-    libffi-dev \
-    libssl-dev \
-    libpulse-dev \
-    build-essential \
-    autoconf \
-    libtool \
-    pkg-config \
-    python-opengl \
-    python-imaging \
-    python-pyrex
+    gcc \
+    python3-pip \
 
 # Update pip
-RUN pip install --upgrade pip
+RUN pip3 install --upgrade pip
 # Install any needed packages specified in requirements.txt
-RUN pip install scrapy textract pybloom botocore
+RUN pip3 install scrapy pdfminer.six botocore pybloom_live requests
```

Comment:
I feel like with 5+ dependencies installed with `pip` you might wanna make a `requirements.txt`